.travis.yml

```diff
@@ -3,10 +3,10 @@ addons:
   apt:
     packages:
       - oracle-java8-installer
-      - oracle-java9-installer
+#      - oracle-java9-installer
```

Comment:
Is it intentional then?

Reply:
Forget it. I saw it is. May be add a comment in the yml file with a link to the issue?

Reply:
On the side I'm tracking the upstream issue, so as soon as it's been fixed I'll re-enable JDK9.